flake.nix

```diff
@@ -17,7 +17,7 @@
             jq
             just
             pnpm
-            # nodejs_20
+            nodejs_20
```

Comment:
Adding this for my local dev purposes, so reqs for this project are in version control.